janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/rest/RestElasticSearchClient.java

```diff
@@ -223,4 +252,19 @@ private Response performRequest(String method, String path, byte[] requestData)
         return response;
     }
 
+    @JsonIgnoreProperties(ignoreUnknown=true)
+    private static final class ClusterInfo {
```

Comment:
The version is obtained from a ES request with default path (e.g. `http://localhost:9200/`). The response is expected to be of the form below and this class is used for unmarshalling to avoid nested `Map` casts. But would AWS ES (or other deployment models) not return a response of this form to this request? I can look into this more.

```javascript
{
  "name" : "X29X896",
  "cluster_name" : "elasticsearch",
  "cluster_uuid" : "ROrFjDOqSIGLBDsZhxFOcA",
  "version" : {
    "number" : "5.2.1",
    "build_hash" : "db0d481",
    "build_date" : "2017-02-09T22:05:32.386Z",
    "build_snapshot" : false,
    "lucene_version" : "6.4.1"
  },
  "tagline" : "You Know, for Search"
}
```